src/esi/Expression.cc

```diff
@@ -97,6 +98,20 @@ stackpop(stackmember * s, int *depth)
     cleanmember(&s[*depth]);
 }
 
+/// push item onto the stack.
```

Comment:
We can make the function description at least slightly useful by mentioning that the item is copied.

```suggestion
/// copies `item` onto the stack
```